src/monk/agent/generator/genSquareVisitor

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Is there a specific reason for changing to Bash ?
Default shell would be Dash, which is lighter and faster, and I do not see Bash specific code in the file.

Reply:
There was some permission issue earlier and I thought this was causing them. I have fixed them now. I will revert it back to `sh`.